scripts/filter_advisories.py

```diff
@@ -0,0 +1,126 @@
+import csv
```

Comment:
Have you considered using any preexisting actions for this?

For example, I found https://github.com/hckops/actions?tab=readme-ov-file#helm-dependencies-action via https://github.com/dependabot/dependabot-core/issues/2237, which seems like it might meet our use case?

Assuming we do need to roll our own, what do you think about extracting this into https://github.com/ministryofjustice/github-actions?

It seems like something that is generally useful but would benefit from more people knowing about it / maintaining it.

Reply:
This suggestion addresses a different part of the [related ticket](https://github.com/ministryofjustice/data-catalogue/issues/32) - namely _keeping the helm chart up to date_. It will also be addressed, in a separate PR.